### ISSUE TITLE
Incorrect translation of special characters

### ISSUE THREAD

**Issue by Kaszanas** [enric1994/bibtexonline#8]
When attepmting to translate a citation from the following source:

```
@inproceedings{10.1145/2187980.2188259,
author = {Kaytoue, Mehdi and Silva, Arlei and Cerf, Lo\"{\i}c and Meira, Wagner and Ra\"{\i}ssi, Chedy},
title = {Watch Me Playing, i Am a Professional: A First Study on Video Game Live Streaming},
year = {2012},
isbn = {9781450312301},
publisher = {Association for Computing Machinery},
address = {New York, NY, USA},
url = {https://doi.org/10.1145/2187980.2188259},
doi = {10.1145/2187980.2188259},
abstract = {"Electronic-sport" (E-Sport) is now established as a new entertainment genre. More and more players enjoy streaming their games, which attract even more viewers. In fact, in a recent social study, casual players were found to prefer watching professional gamers rather than playing the game themselves. Within this context, advertising provides a significant source of revenue to the professional players, the casters (displaying other people's games) and the game streaming platforms. For this paper, we crawled, during more than 100 days, the most popular among such specialized platforms: Twitch.tv. Thanks to these gigabytes of data, we propose a first characterization of a new Web community, and we show, among other results, that the number of viewers of a streaming session evolves in a predictable way, that audience peaks of a game are explainable and that a Condorcet method can be used to sensibly rank the streamers by popularity. Last but not least, we hope that this paper will bring to light the study of E-Sport and its growing community. They indeed deserve the attention of industrial partners (for the large amount of money involved) and researchers (for interesting problems in social network dynamics, personalized recommendation, sentiment analysis, etc.).},
booktitle = {Proceedings of the 21st International Conference on World Wide Web},
pages = {1181–1188},
numpages = {8},
keywords = {e-sport, ranking, popularity prediction, starcraft ii, video game, twitch.tv, social community},
location = {Lyon, France},
series = {WWW '12 Companion}
}
```

This entry generates as follows:
Kaytoue, M., Silva, A., Cerf, L., Meira, W., & Ra\"\issi, C. (2012). Watch Me Playing, i Am a Professional: A First Study on Video Game Live Streaming. In Proceedings of the 21st International Conference on World Wide Web (pp. 1181–1188). Association for Computing Machinery.

```Ra\"\issi``` should be ```Raïssi```

**Comment by enric1994** [comment]
Thanks for your feedback. It seems that we'll need something like [this](https://gist.github.com/ptigas/3552263) but in Javascript.

**Comment by enric1994** [comment]
Fixed in [1a38761](https://github.com/enric1994/bibtexonline/commit/1a387619e35f7025eb13600cb864074008aae803)
